packages/backend/src/index.ts

```diff
@@ -61,10 +61,10 @@ function makeCreateEnv(loadedConfigs: AppConfig[]) {
         client: 'pg',
         useNullAsDefault: true,
         connection: {
-          port: config.getOptionalNumber('backend.POSTGRES_PORT'),
-          host: config.getString('backend.POSTGRES_HOST'),
-          user: config.getString('backend.POSTGRES_USER'),
-          password: config.getString('backend.POSTGRES_PASSWORD'),
+          port: config.getOptionalNumber('backend.POSTGRES.PORT'),
+          host: config.getString('backend.POSTGRES.HOST'),
+          user: config.getString('backend.POSTGRES.USER'),
+          password: config.getString('backend.POSTGRES.PASSWORD'),
```

Comment:
one thing that would be nice to add - if you're up to it, it's optional - is support for e.g. `backend.databaseConnectionString`, that can hold like `'postgres://user:pass@localhost:5432/dbname '`. That string can be consumed directly by Knex. Both that and the postgres-specific separate port/host/user/password could be supported, but just one of them at a time. At that point you may want to break this out into a function of its own that does some validation and throws an error if illegal combinations are used.

Reply:
I added an RFC #1882 to cover some adding some additional shared database code yesterday (which covers this exact feature). Sorry, I didn't realize this PR was already posted at the time.

The Looks my RFC & example code overlaps with this PR. Once feedback if provided, I'm happy to contribute / refactor any additional parts on top of this that make sense.